tests/Doctrine/Tests/OrmTestCase.php

```diff
@@ -171,4 +172,22 @@ protected function getSharedSecondLevelCacheDriverImpl()
 
         return $this->secondLevelCacheDriverImpl;
     }
+
+    public static function assertAttributeEmpty(string $haystackAttributeName, $haystackClassOrObject, string $message = '') : void
```

Comment:
I'd avoid porting these methods to our test case. We should rather find alternative ways to test - compare objects with `assertEquals()`, for example.

Reply:
It was @jwage's idea: https://github.com/doctrine/orm/pull/7688#discussion_r279430589.

I've replaced with assertEquals and methods calling as much as I can, but there are cases that we can't :confused:

Reply:
I see... how many methods are using them?
Can we keep that somewhat contained? Maybe a trait that gets used only when it's **really** necessary?